roofit/roofitcore/src/RooPolyVar.cxx

```diff
@@ -170,21 +177,42 @@ double RooPolyVar::analyticalIntegral(Int_t code, const char* rangeName) const
   R__ASSERT(code==1) ;
 
   const double xmin = _x.min(rangeName), xmax = _x.max(rangeName);
-  const int lowestOrder = _lowestOrder;
   const unsigned sz = _coefList.getSize();
   if (!sz) return xmax - xmin;
```

Comment:
```suggestion
  if (!sz) return _lowestOrder ? xmax - xmin : 0.0;
```
I know that this has nothing to do with your PR technically, but I just realized that this is inconsistent with the definition in `evaluate()`! There should of course also be the different code branches for lowest order equal zero or not, otherwise the integral is not consistent with the values!